stag_node:
  ros__parameters:
    tags:
      "[
        {
          frame: 'tag_7',
          id: 7,
          corners: [
            [ -0.0805,0.0805,0.0 ],
            [ 0.0805,0.0805,0.0 ],
            [ 0.0805,-0.0805,0.0 ]
          ]
        }
      ]"
    bundles: "[]"

# Marker corner order:
# 1          2
#  HD########
#  #ID#  ####
#  ##  ##  ##
#  ####  ####
#  ##########
# 4          3
